Directory.Build.targets

```diff
@@ -91,6 +91,11 @@
       <_UnityPathProp>%(_UnityPathsFoundReversed.Identity)</_UnityPathProp>
     </PropertyGroup>
 
+    <Message
```

Comment:
While setting up the dev environment on Windows, this message would have saved me all the headaches in the world.